### DISCUSSION TITLE
New page, for updates about tShock?

### DISCUSSION THREAD

**Discussion by NaturalBornCamper** [Pryaxis/TShock#1754] [General]
Hello there,

The official tshock page was containing the forums, which will be closed down, and it doesn't seem to have any other info there. Anyone knows if there is an official rss feed or some news posted somewhere so I could set up an alert for news on the progress of tshock 1.4 or other updates in general?

**Comment by alecxvs** [comment]
Updates are being posted to Twitter: https://twitter.com/Pryaxis
